spec/routing/routes_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe 'routes', type: :routing do
```

Comment:
I think you can kill this test. It was useful for you in development but will never be useful again.

The thing with testing for things that _should not_ happen is that it's an infinite set. The app shouldn't Rick Roll people every third thursday but there's no test for that either 😋